flink-libraries/flink-table/src/main/scala/org/apache/flink/table/plan/nodes/datastream/DataStreamGroupWindowAggregate.scala

```diff
@@ -245,6 +264,7 @@ object DataStreamGroupWindowAggregate {
     case SlidingGroupWindow(_, timeField, size, slide)
         if isProctimeAttribute(timeField) && isRowCountLiteral(size) =>
       stream.countWindow(toLong(size), toLong(slide))
+      .trigger(StateCleaningCountTrigger.of(queryConfig, toLong(slide)));
```

Comment:
`slide` is correct here (check `KeyedStream.countWindow(long, long)`).

The default trigger is replaced by a trigger that additional registers a clean up timer. The trigger policy based on counts is still the same.